docs/aquacomputer-d5next-guide.md

```diff
@@ -36,6 +38,20 @@ Aquacomputer D5 Next
 └── +12V voltage          12.06  V
 ```
 
+## Programming the fan speeds
+
+Currently, the pump and optionally connected fan can be set to a fixed duty cycle, ranging from 0-100%.
+
+```
+liquidctl set pump speed 56
+              ^^^^       ^^
+             channel    duty
+```
+
+Valid channel values on the D5 Next are `pump` and `fan`. The hwmon driver is always used first, if it's available and
+exposes `pwmX` and `pwmX_enable` attributes. You can pass in `--direct-access` to bypass it and communicate with the
+device directly.
```

Comment:
I don't think it's necessary to talk about hwmon here, especially since the very next section is about it (and how to bypass it).